builder/bundles.go

```diff
@@ -1123,14 +1122,19 @@ func createVersionsFile(baseDir string, packagerCmd []string) error {
 	return w.Flush()
 }
 
-func fixOSRelease(b *Builder, filename, version string) error {
-
+func updateOSReleaseFile(b *Builder, filename, version string, homeURL string) error {
+	fmt.Println("... updating os-release file")
 	//Replace the default os-release file if customized os-release file found
 	var err error
 	var f *os.File
 	if b.Config.Mixer.OSReleasePath != "" {
 		f, err = os.Open(b.Config.Mixer.OSReleasePath)
 	} else {
+		_, err = os.Stat(filename)
+		if err != nil {
+			err = createOSReleaseFile(filename, homeURL, version)
+			return err
+		}
```

Comment:
Shouldn't we be checking for `os.IsNotExist(err)`
Something like:
```
		if _, err = os.Stat(filename); os.IsNotExist(err) {
			return createOSReleaseFile(filename, homeURL, version)
		}
		if err != nil {
			return err
		}
```

Reply:
yes, will fix it.